terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -615,6 +653,59 @@ module "dms_nomis_ingestor" {
   )
 }
 
+# TODO: DPR-622: Delete when done
+module "dms_nomis_ingestor_full_load" {
+  source                       = "./modules/dms"
+  setup_dms_instance           = local.setup_dms_instance # Disable all DMS Resources
+  enable_replication_task      = local.enable_replication_task # Disable Replication Task
+  name                         = "${local.project}-dms-nomis-ingestor-full-load-${local.env}"
+  vpc_cidr                     = [data.aws_vpc.shared.cidr_block]
+  source_engine_name           = "oracle"
+  source_db_name               = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["db_name"]
+  source_app_username          = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["user"]
+  source_app_password          = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["password"]
+  source_address               = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["endpoint"]
+  source_db_port               = jsondecode(data.aws_secretsmanager_secret_version.nomis.secret_string)["port"]
+  vpc                          = data.aws_vpc.shared.id
+  kinesis_stream_policy        = module.kinesis_stream_ingestor.kinesis_stream_iam_policy_admin_arn
+  project_id                   = local.project
+  env                          = local.environment
+  dms_source_name              = "oracle"
+  dms_target_name              = "kinesis"
+  short_name                   = "nomis"
+  migration_type               = "full-load"
+  replication_instance_version = "3.4.6" # Rollback
+  replication_instance_class   = "dms.t3.medium"
+  subnet_ids                   = [data.aws_subnet.data_subnets_a.id, data.aws_subnet.data_subnets_b.id, data.aws_subnet.data_subnets_c.id]
+
+  enable_experimental_task     = true
```

Comment:
Mark full load DMS task as experimental = true so it uses the experimental settings.